packages/react-native-reanimated/metro-config/index.js

```diff
@@ -0,0 +1,35 @@
+const COLLAPSED_STACK_REGEX = new RegExp(
+  [
+    // For internal usage in the example app
+    "/packages/react-native-reanimated/.+\\.(t|j)sx?$",
+    // When reanimated is installed as a dependency (node_modules)
+    "/node_modules/react-native-reanimated/.+\\.(t|j)sx?$",
+  ]
+    // Make patterns work with both Windows and POSIX paths.
+    .map((pathPattern) => pathPattern.replaceAll("/", "[/\\\\]"))
+    .join("|")
+);
+
+function withReanimated(config) {
```

Comment:
I export `withReanimated` function that takes the merged user config as a parameter instead of exporting a separate config in order not to override the `symbolicator` completely. I want to be able to access user's `symbolicator` (usually the default one from `@react-native/metro-config` or `@expo/metro-config`) to keep the default behavior for logs from `react-native` source code, etc. and just a custom behavior for `react-native-reanimated`.

Reply:
Great work, however I'd rename it to `wrapWithReanimatedMetroConfig` or something like this for clarity.